chore: [TAP 7.1] Update Documentation (#225)
eclipse-tractusx/tractus-x-umbrella: README.md

> 

Reviewer: add legal note to all documents 

## NOTICE

This work is licensed under the [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).

The umbrella helm chart is located in the [charts/umbrella](./charts/umbrella) directory.

## Usage

**Prerequisites & Installation**

> For the prerequisites and installing of the umbrella charts please check the [docs](docs/README.md) section.

Reviewer: ```suggestion
For the prerequisites and installing of the umbrella charts please check the [README](docs/README.md) section.
```

> For the prerequisites and installing of the umbrella charts please check the [docs](docs/README.md) section.

Reviewer: ```suggestion
For the prerequisites and installing of the Umbrella HELM Charts please check the [docs](docs/README.md) section.
```

**Tutorials**

> After installing the umbrella charts, you can refer to the [user guides and tutorials sections](docs/user/guides/README.md) for more information about the dataspace network.

Reviewer: ```suggestion
After installing the Umbrella HELM Charts, you can refer to the [userGuide](docs/user/guides/README.md) for more information about the dataspace network.
```

Reviewer: Is this the correct wording? dataspace network - > CATENA-X dataspace?

## Release Compatibility

> The versions of the [integrated components](docs/user/setup/chart-installation/README.md#available-components) correspond to the **overarching [Release 24.12](https://github.com/eclipse-tractusx/tractus-x-release/blob/24.12/CHANGELOG.md#2412---2024-12-02)**.

Reviewer: ```suggestion
The versions of the [available-components](docs/user/setup/chart-installation/README.md#available-components) correspond to the **overarching [Release 24.12](https://github.com/eclipse-tractusx/tractus-x-release/blob/24.12/CHANGELOG.md#2412---2024-12-02)**.
```

2. **Sandbox Environments**: Create local environments to evaluate and experiment with Catena-X components.
3. **Development**: Provide a unified setup for contributors and developers working on Tractus-X projects.

> 

Reviewer: Is cluster setup not needes any morE?
